example/amagaki.js

```diff
@@ -0,0 +1,21 @@
+const ExamplePlugin = require('./plugins/example/plugin');
```

Comment:
For the purposes of the example (which will serve as reference for people learning about the project), drop the nested `example` folder and just make the file in `/plugins/example.js`. We can then suggest that most projects put their local  plugins in the `plugins` folder (or in `amagaki.js` directly).

Reply:
Also, one last request -- in addition to the sample filter, can you add an example custom YAML type to `/plugins/example.js`? 

I think it should demonstate:

- A custom Nunjucks filter
- A custom YAML type

Reply:
Sounds good, I've moved it up and renamed it.